### DISCUSSION TITLE
My student email is not approved.

### DISCUSSION THREAD

**Discussion by hoonjiyeo** [community/community#120852] [GitHub Education]
### Hi everyone,

My student email address provided by Kyung-In Women's University is not approved. The domain of my email address is office.kiwu.ac.kr, but it seems that only kiwu.ac.kr is approved. It should be noted that office.kiwu.ac.kr is the official domain for student email addresses provided by the university.

**Comment by haululazkiyaa** [comment]
i think github doesnt see your email, but the see your campus ID card, or the documents has related with valid date.

**Comment by samus-aran** [comment]
Hi @hoonjiyeo  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

Hope this helps! Let us know how you get along 👍

**Reply by hoonjiyeo** [reply]
Yes I created a ticket and am waiting for a response, but it's been two weeks and I haven't heard back.
